ngmix/gmix.py

```diff
@@ -908,11 +905,11 @@ def __init__(self, pars, model):
         self._do_init(pars, model)
 
     def _do_init(self, pars, model):
-        self._model = _gmix_model_dict[model]
-        self._model_name = _gmix_string_dict[self._model]
+        self._model = get_model_num(model)
+        self._model_name = get_model_name(self._model)
 
-        self._ngauss = _gmix_ngauss_dict[self._model]
-        self._npars = _gmix_npars_dict[self._model]
+        self._ngauss = get_model_ngauss(self._model)
+        self._npars = get_model_npars(self._model)
```

Comment:
How did our tests miss this?

Reply:
that wasn't wrong.

Using the functions is better because they raise a better error.  Using the dict directly would just raise a KeyError which isn't very informative